include/nestegg/nestegg.h

```diff
@@ -310,6 +314,14 @@ int nestegg_read_packet(nestegg * context, nestegg_packet ** packet);
     @param packet #nestegg_packet to be freed. @see nestegg_read_packet */
 void nestegg_free_packet(nestegg_packet * packet);
 
+/** Query the keyframe status for a given packet.
+    @param packet Packet initialized by #nestegg_read_packet.
+    @retval #NESTEGG_PACKET_KEYFRAME                Packet contains keyframes.
+    @retval #NESTEGG_PACKET_NOT_KEYFRAME            Packet contains no keyframes.
+    @retval #NESTEGG_PACKET_UNKNOWN_KEYFRAME_STATUS Unknown packet keyframe content.
+    @retval -1 Error. */
+int nestegg_packet_keyframe_status(nestegg_packet * packet);
```

Comment:
The prevailing style in nestegg is to use a pointer outparam for values when the function can return an error to avoid mixing error values with other data.

In this case, the function never returns an error anyway, so it's probably simplest to remove the error part and return true/false/unknown values.